pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -866,3 +872,11 @@ impl<T: Config> GetByKey<Domain, Option<[u8; 20]>> for Pallet<T> {
 		DomainHookAddress::<T>::get(domain)
 	}
 }
+
+impl<T: Config> MessageProcessor for Pallet<T> {
+	type Message = T::Message;
+
+	fn process(_: Self::Message) -> DispatchResultWithPostInfo {
+		Ok(PostDispatchInfo::default())
+	}
+}
```

Comment:
Why this is implemented as it is?

Reply:
Actual implementation will be done in the following PR. I can drop this if it's in the way.

Reply:
Also, it's for integration tests. Let's keep it actually and I will change it when implementing the queue in the gateway in the next step.

Reply:
Ok! 👍🏻